### DISCUSSION TITLE
[@types/ws] Type 'Server' is not generic.

### DISCUSSION THREAD

**Discussion by javiercoh** [DefinitelyTyped/DefinitelyTyped#65721] [Issues with a @types package]
Used to work fine but I'm getting the following error now:

`TypeScript: ./..\..\node_modules\@types\ws\index.d.ts:328:18
           Type 'Server' is not generic.`

Please find attached my package.json
[package_share.zip](https://github.com/DefinitelyTyped/DefinitelyTyped/files/11696698/package_share.zip)

Angular CLI: 13.3.11
Node: 16.13.2
Package Manager: npm 8.1.2
OS: win32 x64

Angular: 13.4.0
... common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1303.11
@angular-devkit/build-angular   13.3.11
@angular-devkit/core            13.3.11
@angular-devkit/schematics      13.3.11
@angular/cli                    13.3.11
@angular/pwa                    13.3.11
@schematics/angular             13.3.11
rxjs                            6.6.7
typescript                      5.1.3

[a comment by an automated account is not shown]

**Comment by JansenMoscol** [comment]
@vansergen

**Reply by vansergen** [reply]
Should be fixed in https://github.com/webpack/webpack-dev-server/pull/4899

**Reply by javiercoh** [reply]
thanks I have noticed a new version for webpack-dev-server (4.15.1) being pushed a day ago but when i do npm i still get the same error.

Does version 4.15.1 include the fix?

Please advice if i need to update any dependencies

**Reply by javiercoh** [reply]
npm list throws

```
`-- @angular-devkit/build-angular@13.3.11
  +-- @angular-devkit/build-webpack@0.1303.11
  | `-- webpack-dev-server@4.7.3 deduped
  `-- webpack-dev-server@4.7.3
```


should i try to update everything to latest version?

**Reply by vansergen** [reply]
> should i try to update everything to latest version?

I would give it a shot.
